csp/utils.py

```diff
@@ -95,3 +95,25 @@ def build_policy(config=None, update=None, replace=None, nonce=None):
 
     return '; '.join(['{} {}'.format(k, val).strip()
                       for k, val in policy_parts.items()])
+
+
+# Attempt to support all valid script tag attributes
```

Comment:
On second thought, this isn't quite right. Going to take another stab at this before calling it "done".